frontend/packages/metal3-plugin/src/components/baremetal-nodes/menu-actions.ts

```diff
@@ -0,0 +1,56 @@
+import { Kebab, KebabOption, asAccessReview } from '@console/internal/components/utils';
+import {
+  MarkAsSchedulable,
+  MarkAsUnschedulable,
+} from '@console/app/src/components/nodes/menu-actions';
+import { K8sKind, NodeKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getName } from '@console/shared';
+import { startNodeMaintenanceModal } from '../modals/StartNodeMaintenanceModal';
+import stopNodeMaintenanceModal from '../modals/StopNodeMaintenanceModal';
```

Comment:
Nit: maybe improve consistency e.g. use named or default exports for all modal components :smiley: can be done later.

Reply:
oops 😃, I'll fix this in subsequent PR.